docs/index.md

```diff
@@ -81,6 +81,7 @@ ABM features users have shared that you may want to use in your model
 Mesa Overview <overview>
 tutorials/intro_tutorial
 tutorials/visualization_tutorial
+Migration guide <migration_guide>
```

Comment:
This is great! My overarching reaction is to make it really stand out and make it explicit. 

Keep it in the index but also like around line 57 add it as a hyperlink--- maybe add it up top right under the header to make it easy for users and to stand out -- I would also recommend we are more explicit (e.g. Mesa 2 -> 3 Migration Guide)

Reply:
Maybe start here, and then push it wide when it's (nearly) finished?